src/layer-surface.c

```diff
@@ -5,8 +5,8 @@
 #include "custom-shell-surface.h"
 #include "gtk-wayland.h"
 
-#include "protocol/wlr-layer-shell-unstable-v1-client.h"
-#include "protocol/xdg-shell-client.h"
+#include "wlr-layer-shell-unstable-v1-client.h"
+#include "xdg-shell-client.h"
```

Comment:
My preference would be to not overly pollute the include path and keep protocol/ explicit

Reply:
Looks like that's not easily doable.. meson complains about a subdirectory in a generator's output:

```
src/protocol/meson.build:7:0: ERROR: "outputs" must not contain a directory separator.
```

and wayland-scanner wouldn't create a subdirectory anyway.